docs/02-analyze/02-learning-goals.adoc

```diff
@@ -77,24 +77,24 @@ time or the involved stakeholders. This includes approaches such as:
 
 * Stakeholder analysis and interview,
 * Contextual analysis,
-* Qualitative analysis (e. g., ATAM), in particular, analysis of quality goals,
+* Qualitative analysis (e.g., ATAM), in particular, analysis of quality goals,
 * Analysis of structural deviations between target and current architecture,
 * Quantitative analysis methods and practices, such as:
 * Code metrics (size metrics, coupling, complexity, cohesion),
-* Organisational metrics, such as costs, time, and countable items. Example: error counts and failure rates, development speed, cost per feature, cost per fixed bug, etc.,
-* Runtime metrics, e. g., time and resource demands as well as tools to measure these metrics.
+* Organizational metrics, such as costs, time, and countable items. Example: error counts and failure rates, development speed, cost per feature, cost per fixed bug, etc.,
+* Runtime metrics, e.g., time and resource demands as well as tools to measure these metrics.
```

Comment:
Hier sollte ein Komma am Ende des Satzes sein, da das ja eine Aufzählung ist, und alle Sätze mit einem Komma enden?

Reply:
Nein, da eine neue Liste aufgemacht wird. Hier stimmt auch die Einrückung nicht. Kommt aber in #29